broken-site/broken-site-impl/src/main/java/com/duckduckgo/brokensite/impl/RealBrokenSitePrompt.kt

```diff
@@ -43,16 +43,24 @@ class RealBrokenSitePrompt @Inject constructor(
 
     override suspend fun userDismissedPrompt() {
         if (!_featureEnabled) return
-        if (brokenSiteReportRepository.getDismissStreak() >= brokenSiteReportRepository.getMaxDismissStreak() - 1) {
```

Comment:
Since `brokenSiteReportRepository.getDismissStreak()` is not used anymore, we should remove it along with shared preferences supporting it